PowerEditor/src/MISC/Common/Sorters.h

```diff
@@ -505,7 +505,7 @@ class IntegerSorter : public ISorter
 					{
 						compareResult = a[aNumIndex] - b[bNumIndex];
 
-						// compareResult isn't nesscarily 0
+						// compareResult isn't necessarily 0
```

Comment:
It's not into this PR - The `Files changed` view shown the typo still there.

Reply:
Sorry, I forgot there is a copied and pasted code for ColumnSort and only fixed one typo. It should be fixed now.